### DISCUSSION TITLE
Patching Ingress-controller's OpenSSL-Version

### DISCUSSION THREAD

**Discussion by ibotty** [okd-project/okd#1380] [Q&A]
With the upcoming OpenSSL fix, most likely the OKD router has to be patched as well.  How can this be achieved?  Can one patch the ingress controller to use the custom image?  If so, how can this image be built or will it be built by the OKD project, @vrutkovs?

**Comment by vrutkovs** [comment]
We'll need https://github.com/openshift/router to be updated, built and shipped in OCP nightly, which would automatically land in OKD nightly and be released to OKD stable. The same applies to all other potentially affected images - with a notable exception of `machine-os-content` which we can build ourselves (needs the fixed version to land in F36 repos).

If you would like to update the image sooner feel free to build your own image - https://github.com/okd-project/okd-payload-pipeline/blob/main/buildconfigs/20-haproxy-router.yaml and follow https://github.com/okd-project/okd/blob/master/CONTRIBUTING.md#making-changes to build your own release image
